cityofaustin/atd-moped
# Add status update to Project List View

## Associated issues

https://github.com/cityofaustin/atd-data-tech/issues/9044

Adds status update to columns as well as filtering. 


## Testing
**URL to test:** 

test locally or moped test. 

This branch branches off of the funding source branch, so if you deploy this branch and test, you will see both enhancements on the deployment. 

**Steps to test:**

https://moped-test.austinmobility.io/moped/projects/174?tab=summary

Add a status to the summary page. See it on the project list view. 

Another test to do is to add an internal note on the comments page and check that it does not show up as a status. 

---
#### Ship list
